osgi.specs/docbook/159/service.feature.xml

```diff
@@ -729,6 +707,35 @@ Feature f = builder.build();</programlisting>
         
     </section>
   </section>
+
+
+  <section>
+    <title>Framework Properties</title>
+    <para>
+      When a Feature is launched in an OSGi framework it may be necessary to specify Framework properties. 
+      These can be provided in the Framework Properties extension section of the Feature.
+      The Launcher must be able to satisfy the specified properties. 
+      If it cannot ensure that these are present in the running Framework the launcher must fail.      
+    </para>
+
+    <para>
+      Example:
+      <programlisting>{
+  "id": "org.acme:acmeapp:osgifeature:fw-props:2.0.0",
+  
+  "extensions": {
+    "framework-properties": {
```

Comment:
This should probably be `framework-launching-properties` or `launching-properties`.